tests/users/test_queries.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+from graphql_relay import to_global_id
+
+from django.contrib.auth import get_user_model
+from creator.organizations.factories import OrganizationFactory
+from creator.organizations.models import Organization
+from creator.users.factories import UserFactory
+
+User = get_user_model()
+
+ALL_USERS_ORGANIZATION = """
+query ($org_name: String) {
+  allUsers (organizations_Name: $org_name) {
```

Comment:
the allStudies query uses an id for organization filters - if might be good to do that here as well:

```
allStudies(organization: ID!) 
```

Reply:
yeah, i wasn't sure about this

Reply:
This should be supported by simply plugging in `organization` into the filterset. If not, you can use `GlobalIDMultipleChoiceFilter` to create the filter field.